plugin/storage/es/spanstore/reader_test.go

```diff
@@ -115,7 +115,7 @@ func withSpanReader(fn func(r *spanReaderTest)) {
 		logger:    logger,
 		logBuffer: logBuffer,
 		exporter:  exp,
-		err:       closer(),
+		err:       closer,
```

Comment:
why do you think `err` is an appropriate name?

Reply:
Would it be a `closer` obj?

Reply:
Here's how your struct would look:
```
type spanReaderTest struct {
	client    *mocks.Client
	logger    *zap.Logger
	logBuffer *testutils.Buffer
	exporter  *tracetest.InMemoryExporter
	closer    func() error
	reader    *SpanReader
}
```
From this definition alone, can you explain what `closer` does? Don't say "it shuts down the tracer", because that information is not in this struct definition (whereas all other fields are self-explanatory).

Reply:
That's true. how I can use my golang dumb skill. thanks for your advice as always. it may feel you annoying, I guess.